extensions/amp-experiment/0.1/variant.js

```diff
@@ -34,6 +36,12 @@ export function allocateVariant(win, experimentName, config) {
   assertName(experimentName);
   validateConfig(config);
 
+  // Variant can be overridden from URL fragment.
+  const override = viewerFor(win).getParam(ATTR_PREFIX + experimentName);
+  if (override && config.variants.hasOwnProperty(override)) {
```

Comment:
sorry this comment was based on the code in  https://github.com/ampproject/amphtml/pull/4079. I see that PR is closed. This is fine, ignore my previous comment.